src/Container.php

```diff
@@ -298,6 +302,7 @@ public static function create(): self
                 $container->getMutantCodeFactory(),
             ),
             Differ::class => static fn (): Differ => new Differ(new BaseDiffer(new UnifiedDiffOutputBuilder(''))),
+            SyncEventDispatcher::class => static fn (): SyncEventDispatcher => new SyncEventDispatcher(),
```

Comment:
I had to return this because our classes depend on an interface, not the concrete implementation. So we have to explicitly specify the implementation in this list.